packages/test-interface-criteria/src/criteria/sharedConstraints.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef {import('../types/Criteria').OfferConstraint} OfferConstraint
+ */
+
+/** @type {OfferConstraint} */
+function supportsMinimalProposalFlow(offer) {
+  return Array.isArray(offer.openBookingFlowRequirement)
+    && offer.openBookingFlowRequirement.includes('https://openactive.io/OpenBookingApproval');
```

Comment:
This should be _only_ `https://openactive.io/OpenBookingApproval`, as the test doesn't support attendee details capture or other `openBookingFlowRequirement`s

Reply:
Have resolved this